playbooks/group_vars/trusted_resource_consumers.yml

```diff
@@ -0,0 +1 @@
+acs_play_repository_trusted_urls: "{{ groups['trusted_resource_consumers'] | map('extract', hostvars) | map(attribute='known_urls', default=[]) | flatten }}"
```

Comment:
strictly speaking this var is not specific to repo (even though share and repo are part of the same role...) I would remove the `_repository` pattern.